### PR TITLE
Fixes to /tools and /tools/chrome-devtools + three new tips

### PR DESCRIPTION
**Pull request by pbakaus**
This pull request includes:

- SEO fixes to devtools and tools (i.e. better page titles)
- Fix to counter minification mess with a h1 on the /tools page
- Fix for the tips importer, tested with three new imported tips
- Fixed path to feed in /tools
- Fix in main navigation for DevTools

### CODE REVIEW

**Comment by pbakaus** [comment]
@arthurvr can you proofread the three new tips (markdown files)? They're imported from https://umaar.com/dev-tips/ and I fixed some styling but not much wording. They're pretty small, so should be a quick deal. Thanks!

**Comment by pbakaus** [comment]
@gauntface this pull request needs a lgtm from a writer first, then should be merged by you after a quick look, there are slight infra fixes in it.

**Comment by jpmedley** [comment]
I'm looking at it now.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/5-tricks-to-use-in-the-console-panel.html
+---
+<img src="/web/updates/images/2015-08-10-5-tricks-to-use-in-the-console-panel/five-console-tricks.gif" alt="5 tricks to use in the Console Panel">
+
+#### Use the inspect() command to jump straight to a passed in DOM node
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: 'passed in' should be 'passed-in'.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+rss: false
+layout: update
+published: true
+title: 5 tricks to use in the Console Panel
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: Spell out 5. Do the same in the alt tag below.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/5-tricks-to-use-in-the-console-panel.html
+---
+<img src="/web/updates/images/2015-08-10-5-tricks-to-use-in-the-console-panel/five-console-tricks.gif" alt="5 tricks to use in the Console Panel">
+
+#### Use the inspect() command to jump straight to a passed in DOM node
+
+<pre><code>inspect($('p'))</code></pre>
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: Code needs to be marked like this:
{% highlight javascript %}
inspect($('p'))
{% endhighlight %}

Please change throughout.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/easily-duplicate-dom-nodes.html
+---
+<img src="/web/updates/images/2015-08-10-duplicate-dom-nodes/duplicate-dom.gif" alt="Duplicate DOM nodes">
+
+<em>Or what are essentially cut/copy &amp; paste operations.</em>
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: 'These are essentially' not 'Or what are essentially'.
Replace &amp; with 'and'.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+You can easily change the DOM without having to edit the HTML as a big piece of string.
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: Replace 'as a big piece of string' with 'a giant string'. (The former sounds odd to native speaker ears.)

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+<ul>
+<li>Right click on a node and select <em>Copy</em>.</li>
+<li>You can paste in your code editor, or for prototyping, you can paste the DOM node elsewhere in the DOM tree. The pasted node is inserted as a child of the currently selected node. In the video, I use the left arrow key to jump to the immediate parent opening tag and paste there (which is what I do in most cases).</li>
+</ul>
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: You don't need to fix this, but ordered lists are done like this in markdown:

1. This is a step.
1. This is the next step.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+You can also Cut (right click &gt; Cut, Cmd + x/Ctrl + x) a DOM node and paste - which has the effect of moving the node.
+
+<em>Experiment:</em> Try adding more links in the header/footer of a site using this copy and paste technique and figure out at which point the layout needs improving!
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: No exclamation point.

File: src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/edit-html-in-the-console-panel.html
+---
+<img src="/web/updates/images/2015-08-10-edit-html-in-the-console-panel-of-devtools/console-edit-html.gif" alt="Edit HTML in the Console Panel of DevTools">
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown]: Don't capitalize console panel. Please check throughout.

File: src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+The DOM node context menu you may be familiar with from the Elements panel is present in the Console Panel also.
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown]: This is a lot to digest. Try this

"The DOM node context menu, which you may recognize from the elements panel, is also present  in the console panel."

File: src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+<ol>
+<li>Log a DOM node to the console.</li>
+<li>Right click on it.</li>
+<li>Select Edit as HTML or Edit Text.</li>
+<li>Notice the DOM is updated on the page and also the Elements Panel.</li>
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown]: 'also in the elements panel.' (Added the word 'in'.)

File: src/_langs/en/tools/index.html [hunk 11]
```diff
@@ -16,7 +16,7 @@
   <div class="container">
     <p class="headliner">Get productive.</p>
     <div class="desc">
-      Discover our <h1>tools</h1> and kickstart your development.
+      Discover our&nbsp;<h1>tools</h1>&nbsp;and kickstart your development.
```
Comment (gauntface) [inline src/_langs/en/tools/index.html]: This smells really bad - why is this needed?

Comment (pbakaus) [inline src/_langs/en/tools/index.html]: I wanted "tools" to be the h1 for semantic/SEO purposes, then discovered that our minification process blows away the white-space around the tags. I know this looks ugly :/ I'd be all ears for a better suggestion.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown (hunk 6 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: Done.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown (hunk 5 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: Done.

File: src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown (hunk 4 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown]: Done.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown (hunk 3 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: Done.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown (hunk 2 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: Done.

File: src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown (hunk 1 above)
Comment (pbakaus) [inline src/_langs/en/updates/2015-08-10-5-tricks-to-use-in-the-console-panel.markdown]: Done.

**Comment by umaar** [comment]
Those were a lot of changes :frowning: - sorry about that, I should probably write to a stricter standard. @pbakaus feel free to @ mention me in similar proofreading PRs, the feedback here is very useful.

**Comment by pbakaus** [comment]
@umaar no worries! Will include you next time :)

**Comment by pbakaus** [comment]
@gauntface ping! PTAL.

**Comment by pbakaus** [comment]
@jpmedley could you give a "lgtm" (no merge yet)?
